### DISCUSSION TITLE
Command not found

### DISCUSSION THREAD

**Discussion by balookhan67** [termux/termux-app#3053] [Q&A]
My termux was not showing my payload in my phone storage then someone tell me to use [](https://github.com/Cabbagec/termux-ohmyzsh)
It is still showing me `command not found error`
![IMG_20221022_125216.jpg](https://user-images.githubusercontent.com/64676458/197327655-72f56c87-df23-4e2a-988f-306b22c7ce48.jpg)
Please help me

**Comment by 2096779623** [comment]
`exit` .

**Reply by balookhan67** [reply]
I know the `exit command ` but i have to save my payload in my phone storage
**is there any way to save payload in phone storage** please

**Reply by 2096779623** [reply]
Create a new session.
https://wiki.termux.com/wiki/hacking

**Reply by balookhan67** [reply]
Tbh once i did that and payload got saved in main storage idk how and idk why next time it showed error

`help me bruh`
![FB_IMG_1665395148485.jpg](https://user-images.githubusercontent.com/64676458/197333112-4fd054bd-697d-4bcc-ac46-4ea7a343ddd8.jpg)

**Comment by pressfanksss** [comment]
this is something related to root rights - - and kali linux, try to prescribe
```
exit. 
```

**Reply by pressfanksss** [reply]
```
exit
```
*
